apps/client/src/common/utils/__tests__/regex.test.ts

```diff
@@ -26,8 +26,8 @@ describe('simple tests for regex', () => {
   });
 
   test('startsWithHttp', () => {
-    const right = ['http://test'];
-    const wrong = ['https://test', 'testing', '123.0.1'];
+    const right = ['https://test', 'http://test'];
```

Comment:
Did you get https working?
or did it always work we just did not allow it

Reply:
i have been testing with ontime cloud and it seems fine. can you verify in one of your containers?